packages/framework/assets/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "@shopsys/framework",
+  "version": "9.0.0-alpha.1",
```

Comment:
We talked about 9.0.0-dev version so the package will reflect the version of SSFW. Do I assume properly it's not possible and so you decide to use alpha.1 version instead? 
Or can you explain it to me? I realized I don't know much about releasing node modules.

Reply:
Sorry, my mistake. We can use dev. I fixed it. I think that version system in npm is same as we use yet. https://docs.npmjs.com/about-semantic-versioning